content/shared_directory_xdist/contents.lr

```diff
@@ -0,0 +1,78 @@
+name: Shared directory between master and workers with pytest-xdist
```

Comment:
Sorry, one last thing. IMHO titlecased posts look really nice.

**Shared Directory Between Master and Workers With pytest-xdist**

```no-highlighting
pip install titlecase
python -c "from titlecase import titlecase; print(titlecase('hello world'))"
```